internal/examples/fake/fake.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package fake demonstrates how to use concrete services and fake the
+// interactions with them in tests.
+package fake
+
+import (
+	"fmt"
+	"os"
+
+	"google.golang.org/api/translate/v3"
+)
+
+// TranslateText translates text to the given language using the provided
+// service.
+func TranslateText(service *translate.Service, text, language string) (string, error) {
+	parent := fmt.Sprintf("projects/%s/locations/global", os.Getenv("GOOGLE_CLOUD_PROJECT"))
+	resp, err := service.Projects.Locations.TranslateText(parent, &translate.TranslateTextRequest{
+		TargetLanguageCode: language,
+		Contents:           []string{text},
+	}).Do()
```

Comment:
Consider a separate `req` variable. Sometimes, it can be hard to follow multi-line function calls + declarations.

Reply:
Done